stan/math/prim/scal/err/check_3F2_converges.hpp

```diff
@@ -0,0 +1,80 @@
+#ifndef STAN_MATH_PRIM_SCAL_ERR_CHECK_3F2_CONVERGES_HPP
+#define STAN_MATH_PRIM_SCAL_ERR_CHECK_3F2_CONVERGES_HPP
+
+#include <stan/math/prim/scal/fun/value_of_rec.hpp>
+#include <cmath>
+#include <stdexcept>
+#include <sstream>
+#include <limits>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Check if the hypergeometric function (3F2) called with
+     * supplied arguments will converge, assuming arguments are
+     * finite values.
+     *
+     * @tparam T_a1 Type of a1
+     * @tparam T_a2 Type of a2
+     * @tparam T_a3 Type of a3
+     * @tparam T_b1 Type of b1
+     * @tparam T_b2 Type of b2
+     * @tparam T_z Type of z
+     *
+     * @param function Name of function ultimately relying on 3F2 (for error
+     &   messages)
+     * @param a1 Variable to check
+     * @param a2 Variable to check
+     * @param a3 Variable to check
+     * @param b1 Variable to check
+     * @param b2 Variable to check
+     * @param z Variable to check
+     *
+     * @throw <code>domain_error</code> if 3F2(a1, a2, a3, b1, b2, z)
+     *   does not meet convergence conditions, or if any coefficient is NaN.
+     */
+    template <typename T_a1, typename T_a2, typename T_a3, typename T_b1,
+      typename T_b2, typename T_z>
```

Comment:
The continued line `typename` should be lined up under the first `typename` on the previous line.